committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+  - Foster an inclusive culture
+- Ensuring continued area health through proactively
+  - Triaging issues
+  - Reviewing PRs
+  - Participation in monitoring of test health
+  - Fixing tests
+  - Contributing features
+  - Refactoring and simplifying codebase
+  - Participating in design discussions and providing feedback
+- Taking technical ownership of areas governed by the SIG
+  - Required: Ensuring health of code authored
+  - Optional: Ensuring health of code authored by other members
+  - Health includes
+    - contributions adhere to criteria defined by style and contribution guides
+    - code correctness is tested through automation
+    - tests remain passing
+    - issues are resolved and patched if needed
+    - code remains simple and approachable to others
+      - self documenting code through function and variable naming
+      - explicit documentation
+  - Making technical decisions with other members
+- Responding to issues facing the SIG
+  - Fixing critical bugs
+
+### Externally facing responsibilities
+
+- Acting as ambassadors to the SIG through
+  - Optional: Responding to messages on communication channels
+  - Optional: Participating in events such as meetups
+
+#### Selecting members
+
+Members should show a commitment to the SIG before becoming members.
+This maybe a verbal commitment - e.g. I plan to work in this area
+full time for the next 3 months - or something demonstrated over time
+through contributions in PRs, issue triage, test triage, operations,
+etc.
+
+Members are expected to be responsive to issues for sub areas
+they own within the SIG.  Members are expected to help other
+members in the SIG (e.g. transfer knowledge, help debug, assist with
+operational tasks, answer questions, etc).
+
+Membership is provisional until the member has shown sustained
+contributions for 3 months.  Provisional members may provide
+feedback in decision making, but are not deciders in the decision
+itself.
+
+## Processes and tools
+
+The following section outlines various processes used by the SIG.
+
+### Decision making within the SIG
+
+It is critical to a SIG that the decision making processes is healthy.
+A good decision making process has the following properties:
+
+- Decision making process is unambiguously understood by everyone involved
+- Decisions are resolved instead of languishing and defaulting to no decision
+- Decisions are a commitment and won't waffle after having been made
+
+#### Who, What, Where, When, How
+
+In cases where a proposal author is seeking acceptance from the
+community it is important to have a clearly defined process for
+communication and resolution of the proposal.
+
+While it is not necessary for all decisions to go through the lazy
+consensus through the full SIG, it is important that such a process
+exists for decisions that may affect many members of the SIG.
+
+When using [lazy consensus][lazy-consensus], all proposals must be published to a visible
```

Comment:
An aspect of lazy consensus is a timeline for reviews.  It might be good to have minimum times for issues to be considered for lazy consensus.  The more important the issue, the longer it should be be open for people to comment.  It would suck to have a critical decision made while you were out on vacation.

Reply:
Added something about this.  Made the recommended minimum 5 business days, but also added that quorum of area OWNERs should be present for any decision in an area.